frontend/dispatcher.lua

```diff
@@ -1079,7 +1079,9 @@ function Dispatcher:_showAsMenu(settings, exec_props)
             font_size = 22,
             callback = function()
                 UIManager:close(quickmenu)
-                Dispatcher:execute(settings, { qm_show = false })
+                UIManager:nextTick(function()
+                    Dispatcher:execute(settings, { qm_show = false })
```

Comment:
Actually why is this Dispatcher and not self, anyway?

Reply:
Maybe @yparitcher emphasizes that there are no instances but one and only mighty Dispatcher?

Reply:
Haha, maybe. Anyway, I don't mind.

Reply:
IIRC when i created dispatcher it was to show that it is a singleton, (also that was my first big contribution to koreader, so was still figuring out lua) this style is throughout the whole dispatcher, I didn't use `self` much